src/types/usersAttributes.ts

```diff
@@ -0,0 +1,16 @@
+import {
```

Comment:
I'd prefer to name file as models, to have all model types after that at this file

Reply:
done